tests/ViewComposers/TestCountryLocaleViewComposer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace PodPoint\Countries\Tests\ViewComposers;
+
+use PodPoint\Countries\ViewComposers\CountryLocaleViewComposer;
+use Illuminate\View\View;
+use Illuminate\Config\Repository;
+use PHPUnit\Framework\TestCase as PHPUnitTestCase;
+
+class TestCountryLocaleViewComposer extends PHPUnitTestCase
+{
+    /**
+     * Makes sure every methods of the view composer are called properly.
+     */
+    public function testCompose()
+    {
+        /** @var Repository|\PHPUnit_Framework_MockObject_MockObject $configMock */
+        $configMock = $this->getMockBuilder(Repository::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['get'])
+            ->getMock();
+
+        $countryLocaleViewComposer = new CountryLocaleViewComposer($configMock);
+
+        /** @var View|\PHPUnit_Framework_MockObject_MockObject $viewMock */
+        $viewMock = $this->getMockBuilder(View::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['with'])
+            ->getMock();
+
+        $viewMock->expects($this->once())
+            ->method('with');
+
+        $configMock->expects($this->once())
```

Comment:
I wouldn't have bothered to mock this config stuff.

Reply:
Okay I'll create a new instance of Repository for my next test then.